src/App.module.scss

```diff
@@ -1,2 +1,13 @@
 .appContainer {
+  .addMovieButton {
+    position: absolute;
+    top: -60px;
```

Comment:
If I run via test in package.json:
![image](https://github.com/user-attachments/assets/a38db43c-ea91-49ae-8718-d24827ba78a5)


if just `jest --coverage` 
![image](https://github.com/user-attachments/assets/c9aa2b19-f36f-4d69-9863-b7280b708ac0)

Reply:
Fixed, please try again